openstack/workflow/v2/workflows/results.go

```diff
@@ -0,0 +1,56 @@
+package workflows
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// CreateResult is the response of a Post operations. Call its Extract method to interpret it as a Workflow.
+type CreateResult struct {
+	commonResult
+}
+
+// Extract helps to get a Workflow struct from a Get function.
+func (r commonResult) Extract() (*Workflow, error) {
```

Comment:
If I'm understanding correctly, "create" will return a list (https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/workflow.py#L169-L173), even if there's only one workflow, but GET will return a single workflow?

If so, then maybe have a `GetResult` and a `CreateResult` rather than one inheriting from the other - might add better separation.

Also, slight nit, but this shouldn't be in this PR. No doubt this was accidental, though, so it's OK to keep it. 😄